### PR TITLE
Update amp-sticky-ad.md

### PR DESCRIPTION
**Pull request by zhouyx**

### CODE REVIEW

File: extensions/amp-sticky-ad/amp-sticky-ad.md [hunk 1]
```diff
@@ -13,3 +13,52 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+  </tr>
+</table>
+
+## Behavior
+
+- There can be only one `<amp-sticky-ad>` in an AMP document. The `<amp-sticky-ad>` should only have one direct child of `<amp-ad>`.
+- The sticky will appear on the bottom of a page.
+- The default height of sticky-ad is 100px.
+- The max-height of the sticky-ad is 100px, if the height exceeds 100px then the height would be 100px and overflow content will be hidden.
+- The width of the sticky-ad is set to 100% ussing CSS and cannot be overrided.
```
Comment (dvoytenko) [inline extensions/amp-sticky-ad/amp-sticky-ad.md]: "overridden"

File: extensions/amp-sticky-ad/amp-sticky-ad.md [hunk 2]
```diff
@@ -13,3 +13,52 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>NODISPLAY</td>
+  </tr>
+</table>
+
+## Behavior
+
+- There can be only one `<amp-sticky-ad>` in an AMP document. The `<amp-sticky-ad>` should only have one direct child of `<amp-ad>`.
+- The sticky will appear on the bottom of a page.
+- The default height of sticky-ad is 100px.
```
Comment (dvoytenko) [inline extensions/amp-sticky-ad/amp-sticky-ad.md]: Default height is not `100px`, but rather whatever ad needs up to `100px`, right?

**Comment by dvoytenko** [comment]
LGTM

File: extensions/amp-sticky-ad/amp-sticky-ad.md [hunk 3]
```diff
@@ -13,3 +13,52 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+  </tr>
+</table>
+
+## Behavior
+
+- There can be only one `<amp-sticky-ad>` in an AMP document. The `<amp-sticky-ad>` should only have one direct child of `<amp-ad>`.
+- The sticky will appear on the bottom of a page.
+- The height of the sticky-ad is whatever its child needs up to its max-height.
+- The max-height of the sticky-ad is 100px, if the height exceeds 100px then the height would be 100px and overflow content will be hidden.
+- The width of the sticky-ad is set to 100% ussing CSS and cannot be overridden.
```
Comment (rudygalfi) [inline extensions/amp-sticky-ad/amp-sticky-ad.md]: spelling "using"

File: extensions/amp-sticky-ad/amp-sticky-ad.md [hunk 4]
```diff
@@ -13,3 +13,52 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+- The sticky-ad will display after scroll one viewport height from top provided there is at-least one more viewport of content available.
```
Comment (rudygalfi) [inline extensions/amp-sticky-ad/amp-sticky-ad.md]: "there is at least" (no hyphen)

File: extensions/amp-sticky-ad/amp-sticky-ad.md [hunk 5]
```diff
@@ -13,3 +13,52 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+- Swipe to dismiss (TBD)
+
+Example:
+```html
+<amp-sticky-ad layout="nodisplay">
```
Comment (rudygalfi) [inline extensions/amp-sticky-ad/amp-sticky-ad.md]: the indentation in this block is a bit messed up. I think `<amp-ad>` should only be indented 2 spaces and there's an extra space before the closing `</amp-sticky-ad>`.

**Comment by rudygalfi** [comment]
A couple small stylistic comments.

**Comment by zhouyx** [comment]
@rudygalfi 
Thanks for point them out!  fixed, PTAL

**Comment by rudygalfi** [comment]
LGTM, thanks!

**Pull request merged**